src/components/graph/graph.config.js

```diff
@@ -113,6 +113,7 @@
  *   - 'wye'
  *
  * **[note]** react-d3-graph will map this values to [d3 symbols](https://github.com/d3/d3-shape#symbols)
+ * @param {Function} [node.viewGenerator=undefined] - 🔍🔍🔍 function that receives a node and returns a JSX view.
  * <br/>
  * @param {Object} link link object is explained in the next section. ⬇️
```

Comment:
Please add documentation on the new function that is being exposed in the `react-d3-graph` component API.